golden_bridge.rb

```diff
@@ -0,0 +1,61 @@
+class Array
+  def convert_to_integer
```

Comment:
@Murzic In my practice I had not so much work with custom methods that were added in base class. In big application sometimes it was hard to understand how it's working because custom methods had no docs. 
In this case I had a chance to implement like I did. In big applications I would do this conversion:

```
def prepare_data(received_array)
  convert_to_integer(received_array).quicksort
end

def convert_to_integer(array)
  return [] if array.empty?
  array.map(&:to_i)
end
```